test/ui-test/lightspeedUiTestExplorer.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  ActionsControl,
+  ActivityBar,
+  By,
+  ContextMenu,
+  EditorView,
+  InputBox,
+  ModalDialog,
+  SideBarView,
+  VSBrowser,
+  ViewControl,
+  ViewSection,
+  WebView,
+  WebviewView,
+  Workbench,
+} from "vscode-extension-tester";
+import {
+  getFixturePath,
+  expectNotification,
+  getModalDialogAndMessage,
+  getWebviewByLocator,
+  sleep,
+  updateSettings,
+  workbenchExecuteCommand,
+} from "./uiTestHelper";
+import { WizardGenerationActionType } from "../../src/definitions/lightspeed";
+import { PlaybookGenerationActionEvent } from "../../src/interfaces/lightspeed";
+import { expect } from "chai";
+import axios from "axios";
+
+export function lightspeedUIExplorerTest(): void {
```

Comment:
You can just remove the lightspeedUIExplorerTest that wraps the test and rename the file to have a `Test.ts` suffix.

Reply:
Thanks.. I wrote this test before your PR was merged and I've obviously forgotten to update this bit...

.. makes me worried that the new test hasn't actually ran yet... :fearful: